### ISSUE TITLE
scraping: mavlink version not determined properly

### ISSUE THREAD

**Issue by jbohren** [rosindex/rosindex#138]
See missing dep here: http://rosindex.github.io/p/mavros/github-mavlink-mavros/#indigo-deps

```
[43.39%] Scraping github-mavlink-mavlink-gbp-release...
 --- no version for mavlink instance: github-mavlink-mavlink-gbp-release distro: jade
 --- no version for mavlink instance: github-mavlink-mavlink-gbp-release distro: indigo
 --- no version for mavlink instance: github-mavlink-mavlink-gbp-release distro: hydro
 --- no version for mavlink instance: github-mavlink-mavlink-gbp-release distro: groovy
 --- no version for mavlink instance: github-mavlink-mavlink-gbp-release distro: fuerte
 --- no version for mavlink instance: github-mavlink-mavlink-gbp-release distro: electric
 --- no version for mavlink instance: github-mavlink-mavlink-gbp-release distro: diamondback
```

**Comment by jbohren** [comment]
It seems like this is a problem with `release` repos listed as `doc` repos. Probably the version extraction isn't working because these things put forward slashes in branch and tag names.

**Comment by jbohren** [comment]
Fixed in #143
